port/main.c

```diff
@@ -167,11 +175,12 @@ static void mainLoop(void)
 			break;
 
 		case mtWrite:
+#if HAVE_PF
 				if (msg.i.io.oid.id == pf_oid.id)
 					msg.o.io.err = readPf(msg.i.data, msg.i.size, msg.i.io.offs);
 				else
+#endif
 					msg.o.io.err = -EINVAL;
```

Comment:
maybe it could be set always before the ifdef? because the current code seems less-readable for me:
```c
    msg.o.io.err = -EINVAL;
#if defined(HAVE_PF)
    if (msg.i.io.oid.id == pf_oid.id)
	msg.o.io.err = readPf(msg.i.data, msg.i.size, msg.i.io.offs);
#endif
```
If that would be my code, I would request at least for braces to be added, eg:
```c
#if defined(HAVE_PF)
    if (msg.i.io.oid.id == pf_oid.id) {
	msg.o.io.err = readPf(msg.i.data, msg.i.size, msg.i.io.offs);
    } else
#endif
    {
	msg.o.io.err = -EINVAL;
    }
```